garak/generators/openai.py

```diff
@@ -58,6 +58,29 @@
     # "ada",  # shutdown https://platform.openai.com/docs/deprecations
 )
 
+context_lengths = {
```

Comment:
In a perfect world, we'd pull these via the openai api, but IIRC, this is not exposed there.